apps/prairielearn/elements/pl-dataframe/pl-dataframe.mustache

```diff
@@ -2,8 +2,8 @@
 <div id="pl-dataframe-{{uuid}}" class="card mb-4">
   <div class="card-header">
     <ul class="nav nav-tabs card-header-tabs" role="tablist">
-      <li class="nav-item" role="presentation"><a class="nav-link active" data-bs-toggle="tab" href="#table-{{uuid}}">Table</a></li>
```

Comment:
This is an accessibility fix that wasn't caught since the `pl-dataframe` element wasn't included in those checks.